### PR TITLE
Docs: Update "Securing Kubernetes" for Ingress Controller

### PR DESCRIPTION
**Pull request by alexfornuto**
## Summary

Updates [Securing Kubernetes](https://pomerium.com/guides/kubernetes.html) to continue from our Helm-based install instructions, and set a route as an Ingress.

## Related issues

Closes https://github.com/pomerium/docs/issues/7

## Checklist

- [ ] reference any related issues
- [ ] updated docs
- [ ] updated unit tests
- [ ] updated UPGRADING.md
- [ ] add appropriate tag (`improvement` / `bug` / etc)
- [ ] https://github.com/pomerium/pomerium/pull/2667 is already merged (this PR may require a rebase afterwards)
- [ ] ready for review

### CODE REVIEW

File: docs/guides/kubernetes.md [hunk 1]
```diff
@@ -107,87 +98,41 @@ subjects:
...
-  namespace: default
-  name: pomerium
-  labels:
-    app: pomerium
+  name: k8s
+  annotations:
+    cert-manager.io/issuer: pomerium-issuer
+    ingress.pomerium.io/policy: '[{"allow":{"and":[{"domain":{"is":"pomerium.com"}}]}}]'
+    ingress.pomerium.io/secure_upstream: true
+    ingress.pomerium.io/allow_spdy: true
```
Comment (alexfornuto) [inline docs/guides/kubernetes.md:114]: This annotation does not yet exist, but is required for the route to work.

File: docs/guides/kubernetes.md [hunk 2]
```diff
@@ -202,7 +147,9 @@ The policy should be a base64-encoded block of yaml:
   kubernetes_service_account_token: "..." #$(kubectl get secret/"$(kubectl get serviceaccount/pomerium -o json | jq -r '.secrets[0].name')" -o json | jq -r .data.token | base64 -d)
 ```
 
-Applying this configuration will create a Pomerium deployment and service within kubernetes that is accessible from `*.localhost.pomerium.io:30443`.
+:::
+
+Applying this configuration change will create a Pomerium route within kubernetes that is accessible from `*.localhost.pomerium.io:30443`.
 
 ## Kubectl
```
Comment (alexfornuto) [inline docs/guides/kubernetes.md:155]: @travisgroth the section below says it's installing something kubernetes-specific, but it's really the pomerium-cli, right? Should I rewrite it to name it as such, link to our docs on it, and reference our existing install instructions for it?

**Review by alexfornuto — COMMENTED** [review]
This is still a work in progress.

File: docs/guides/kubernetes.md (hunk 2 above)
Comment (travisgroth) [reply docs/guides/kubernetes.md:161]: It's installing `pomerium-cli` - which we should update the instructions for - but then configuring `kubectl` to use it as an auth helper.  Maybe rename to

  ```
## Configure kubectl
```

And link to install instructions for the cli.

**Comment by alexfornuto** [comment]
Closing in favor of #2758 to avoid dealing with merge conflicts and rebasing.

**Pull request closed without merging**